### DISCUSSION TITLE
Feature Request - Windows UI Daemon client/start minimized to traybar

### DISCUSSION THREAD

**Discussion by wcbutler** [Chia-Network/chia-blockchain#14360] [Ideas]
I've been running the software via FireDaemon to keep it out of my way, before that I had it on a secondary Win 11 desktop. Would be really nice to have the client minimize to traybar especially if that is available on startup. I know I can run the CLI to farm but I'd like to have the UI there to check but have it load on my main workstation in the background in this way.

**Comment by BrandtH22** [comment]
Hey @wcbutler  , exciting news, the upcoming 2.3 release will have the ability to keep services running in the background when shutting down the GUI 🥳 

I know this does not have a traybar icon but it is a big step towards that end

**Comment by BrandtH22** [comment]
2.3 was just released with this running services in the background feature:

You can download 2.3 from here: https://www.chia.net/downloads/
This overviews how to use the background services feature: https://docs.chia.net/using-the-gui/#run-chia-services-in-the-background

Let us know if any questions arise and since this feature request is a duplicate of #8462 I am going to close this ticket and use the other one for further updates 👍
